config/zones/TR.yaml

```diff
@@ -120,6 +124,12 @@ emissionFactors:
       datetime: '2021-01-01'
       source: INCER ACV
       value: 26.15
+    geothermal:
+      datetime: '2020-01-01'
+      source: Akın, Serhat, Yasemin Örücü, and Thrainn Frıdrıksson. "Characterizing the declining CO2 emissions from \n
+      Turkish geothermal power plants." (2020)
+      url: https://pangea.stanford.edu/ERE/db/GeoConf/papers/SGW/2020/Akin.pdf
```

Comment:
url is not a valid key for this config, we only use source to link to the research paper

Reply:
In [KR.yaml](https://github.com/electricitymaps/electricitymaps-contrib/blob/master/config/zones/KR.yaml), there is a url linked for unknown using the key `_url`